crates/astria-composer/src/searcher/mod.rs

```diff
@@ -181,31 +133,26 @@ impl Searcher {
             .map(|(chain_id, collector)| (chain_id.clone(), collector.subscribe()))
             .collect();
 
-        // connect to sequencer node
-        let sequencer_client = SequencerClient::new(&cfg.sequencer_url)
-            .wrap_err("failed constructing sequencer client")?;
-
         let (status, _) = watch::channel(Status::default());
 
-        // create signing key for sequencer txs
-        let mut private_key_bytes: [u8; 32] = hex::decode(cfg.private_key.expose_secret())
-            .wrap_err("failed to decode private key bytes from hex string")?
-            .try_into()
-            .map_err(|_| eyre!("invalid private key length; must be 32 bytes"))?;
-        let sequencer_key =
-            SigningKey::try_from(private_key_bytes).wrap_err("failed to parse sequencer key")?;
-        private_key_bytes.zeroize();
+        // create channel for sending bundles to executor
+        let (executor_tx, executor_rx) = mpsc::channel(BUNDLE_CHANNEL_SIZE);
```

Comment:
IMO remove the const here, just put the number here.

Reply:
https://github.com/astriaorg/astria/pull/294#discussion_r1323202939